doc/en/source/https.rst

```diff
@@ -262,28 +265,38 @@ Change the IP configuration accordingly
 
 Configuration file need to be modified at the following places with IP addresses  according to user own environment.
 
-- Change PEP Proxy host-port and container port in docker-compose.idm.yml file.
-
-- Change the IdM config file at following places as per the environment.
+- Change the IdM config file (idm_config.js) at following places as per the environment.
 
 
 .. code-block:: console
 
         
         config.port = 3000;
-        config.host = "http://<IdM IP>:" + config.port;
-
-        config.database = {
-            host: "localhost",
-            password: "idm",
-            username: "root",
-            database: "idm",
-            dialect: "mysql",
-            port: undefined
-        };
+        config.host = '<IdM_IP>';   // eg; config.host = '180.179.214.215';
+
+Note: IdM_IP denotes the IP of cloud node in this case, if IdM instance is to be set on platform other than cloud; user must provide IP of that platform.
```

Comment:
why not use the host-name in the same docker-compose file? For the multiple docker images, the host name could be used to access the other services in the same docker-compose file. In this case, we can avoid the IP configuration. Not sure if this is the case here.